lib/modules/@apostrophecms/notifications/index.js

```diff
@@ -49,14 +49,27 @@ module.exports = {
             pulse: pulse,
             id: id
           }]));
-        },
-        async dismiss(req) {
-          const _id = self.apos.launder.id(req.body._id);
-          await self.db.deleteMany({ _id: _id });
         }
       }
     };
   },
+  restApiRoutes: (self, options) => ({
+    getOne(req, _id) {
+      return self.find(req, { displayingIds: [_id] });
+    },
+    post(req) {
+      return self.trigger(req, req.body.message, req.body.options || {});
```

Comment:
The old `trigger` apiRoute's validation code should be moved here, and the old trigger route should be removed.